src/brogue/IO.c

```diff
@@ -2596,6 +2596,11 @@ void executeKeystroke(signed long keystroke, boolean controlKey, boolean shiftKe
                 enableEasyMode();
             //}
             break;
+        case PRINTSCREEN_KEY:
+            if (takeScreenshot()) {
+                flashTemporaryAlert("You take a new screenshot", 2000);
```

Comment:
Being really picky, could you change `"You take a new screenshot"` to `" Screenshot saved in save directory "` (with the surrounding spaces)? The reasons being 1) we should try to tell the player where their screenshot is and 2) all other temporary alerts have a space on both sides

Reply:
Yes of course ! As usual, for display text I will ask you :)

Reply:
When you said "... in **save directory**". You want to replace **save directory** to the CWD ?

Reply:
No, that would probably be too long, just verbatim "save directory".